README.md

```diff
@@ -1,2 +1,34 @@
-## Dev-Mountain Solidity Contract
+# **Dev. Mountain Tech Festival(Thailand) 2022 - Solidity Contract**
 
+
+## What is Dev. Mountain Tech Festival  
+
+Dev. Mountain Tech Festival is Thailand tech events that setup at
+18-20 March 2022 Toscana Valley Khao Yai.This event is have many top-tier Thai developers as speaker and have many different topic for developers or anyone who interested in technology
+Other way this tech events is have the blockchain hackathon inside it 
```

Comment:
in additional, this tech event also has blockchain hackathon inside it

Reply:
Fixed here -> https://github.com/LAzyDev-101/Dev-mountain-hackthon-contract/pull/5/commits/a48f2fd7df405b91e50f76963fa8b52e6e9e3465